Flux provider crashed

I added it the creation of the bootstrap as it is explained on the github-ssh example and when i applied this was the output:
`│ Error: Request cancelled
│
│ The plugin6.(*GRPCProvider).ApplyResourceChange request was cancelled.
╵

Stack trace from the terraform-provider-flux_v0.25.3 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x103524e7c]

goroutine 28 [running]:
github.com/fluxcd/terraform-provider-flux/internal/provider.(*providerResourceData).GetKubernetesClient(0x14000478128?)
	github.com/fluxcd/terraform-provider-flux/internal/provider/provider_resource_data.go:64 +0x2c
github.com/fluxcd/terraform-provider-flux/internal/provider.(*bootstrapGitResource).Create(0x1400061c5c0, {0x103cc8ee8, 0x14000951d40}, {{{{0x103cd22c0, 0x14000965410}, {0x1039f6e80, 0x14000964cf0}}, {0x103cd5c00, 0x1400014a5a0}}, {{{0x103cd22c0, ...}, ...}, ...}, ...}, ...)
	github.com/fluxcd/terraform-provider-flux/internal/provider/resource_bootstrap_git.go:335 +0x130
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0x140005314a0, {0x103cc8ee8, 0x14000951d40}, 0x14000543370, 0x14000543310)
	github.com/hashicorp/terraform-plugin-framework@v1.1.1/internal/fwserver/server_createresource.go:97 +0x418
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0x140007534d0?, {0x103cc8ee8, 0x14000951d40}, 0x14000456d20, 0x140005434d0)
	github.com/hashicorp/terraform-plugin-framework@v1.1.1/internal/fwserver/server_applyresourcechange.go:54 +0x378
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ApplyResourceChange(0x140005314a0, {0x103cc8ee8?, 0x14000951bf0?}, 0x14000456c80)
	github.com/hashicorp/terraform-plugin-framework@v1.1.1/internal/proto6server/server_applyresourcechange.go:52 +0x314
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0x140007f00a0, {0x103cc8ee8?, 0x14000951410?}, 0x14000120cb0)
	github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov6/tf6server/server.go:816 +0x3b8
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0x103bd33e0?, 0x140007f00a0}, {0x103cc8ee8, 0x14000951410}, 0x14000120bd0, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:385 +0x164
google.golang.org/grpc.(*Server).processUnaryRPC(0x1400001a1e0, {0x103cd2fb8, 0x14000582680}, 0x1400094e360, 0x1400021ffb0, 0x104c09ae0, 0x0)
	google.golang.org/grpc@v1.51.0/server.go:1340 +0xb38
google.golang.org/grpc.(*Server).handleStream(0x1400001a1e0, {0x103cd2fb8, 0x14000582680}, 0x1400094e360, 0x0)
	google.golang.org/grpc@v1.51.0/server.go:1713 +0x80c
google.golang.org/grpc.(*Server).serveStreams.func1.2()
	google.golang.org/grpc@v1.51.0/server.go:965 +0x84
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.51.0/server.go:963 +0x278

Error: The terraform-provider-flux_v0.25.3 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

make: *** [apply] Error 1`

--- a/internal/provider/provider_resource_data.go
+++ b/internal/provider/provider_resource_data.go
@@ -61,6 +61,9 @@ func NewProviderResourceData(ctx context.Context, data ProviderModel) (*provider
 }
 
 func (prd *providerResourceData) GetKubernetesClient() (client.WithWatch, error) {
+	if prd.rcg == nil {
+		return nil, fmt.Errorf("Kubernetes client cannot be created without any Kubernetes provider configuration")
+	}
 	kubeClient, err := utils.KubeClient(prd.rcg, &runclient.Options{})
 	if err != nil {
 		return nil, err

